rotkehlchen/chain/evm/decoding/balancer/mixins.py

```diff
@@ -0,0 +1,111 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.decoding.balancer.constants import CPT_BALANCER_V1, CPT_BALANCER_V2
+from rotkehlchen.chain.evm.decoding.utils import maybe_reshuffle_events
+from rotkehlchen.chain.evm.structures import EvmTxReceiptLog
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.types import EvmTransaction
+
+if TYPE_CHECKING:
+    from rotkehlchen.history.events.structures.evm_event import EvmEvent
+
+
+class BalancerCommonAccountingMixin:
+    def __init__(self, counterparty: str) -> None:
+        self.counterparty = counterparty
+
+    def _check_deposits_withdrawals(
+            self,
+            transaction: EvmTransaction,  # pylint: disable=unused-argument
+            decoded_events: list['EvmEvent'],
+            all_logs: list[EvmTxReceiptLog],  # pylint: disable=unused-argument
+    ) -> list['EvmEvent']:
+        """Check for accounting in balancer that the deposits/withdrawals events have the needed information to process them during accounting."""  # noqa: E501
```

Comment:
Hmm this sentence does not make sense. What do you want to write?

Reply:
fixed it.